skbio/diversity/beta/_fast_unifrac.py

```diff
@@ -0,0 +1,390 @@
+# ----------------------------------------------------------------------------
+# Copyright (c) 2013--, scikit-bio development team.
+#
+# Distributed under the terms of the Modified BSD License.
+#
+# The full license is in the file COPYING.txt, distributed with this software.
+# ----------------------------------------------------------------------------
+
+from __future__ import absolute_import, division, print_function
+
+import numpy as np
+
+from skbio.util._decorator import experimental
+from ._unifrac import _validate
+from skbio.diversity._fast_base import counts_and_index
+from skbio.diversity._fast_base_cy import tip_distances
+
+
+def unifrac(m, i, j):
```

Comment:
Unclear to me here, but not opposed
On Sep 9, 2015 3:26 PM, "mortonjt" <notifications@github.com> wrote:

> In skbio/diversity/beta/_fast_unifrac.py
> <https://github.com/biocore/scikit-bio/pull/1040#discussion_r39035564>:
>
> > +# Distributed under the terms of the Modified BSD License.
> > +#
> > +# The full license is in the file COPYING.txt, distributed with this software.
> > +# ----------------------------------------------------------------------------
> > +
> > +from __future__ import absolute_import, division, print_function
> > +
> > +import numpy as np
> > +
> > +from skbio.util._decorator import experimental
> > +from ._unifrac import _validate
> > +from skbio.diversity._fast_base import counts_and_index
> > +from skbio.diversity._fast_base_cy import tip_distances
> > +
> > +
> > +def unifrac(m, i, j):
>
> Don't forget the experimental tags :)
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/biocore/scikit-bio/pull/1040/files#r39035564>.
>